packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -4534,6 +4534,191 @@ export class WalletService {
     return cb(null, data);
   }
 
+  moonpayGetKeys(req) {
```

Comment:
Let's go ahead and make this a `private` function

Reply:
Done